UnityProject/Assets/Scripts/Core/UniversalObjectPhysics.cs

```diff
@@ -1519,7 +1519,7 @@ public void FlyingUpdateMe()
 					{
 						if (isServer)
 						{
-							if ((UniversalObjectPhysics) bump == this) continue;
+							if (bump as UniversalObjectPhysics == this) continue;
```

Comment:
why is this different?

Reply:
it does some automatic null check in the background i believe

Reply:
ah just remembered, the first one will throw, the second doesn't throw but leaves the value as null

Reply:
apparently throws an error if it can't cast with the other one